keyboards/murcielago/info.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
This info.json produces:
![image](https://user-images.githubusercontent.com/16520359/81850863-7a4cd780-9550-11ea-9f70-fd0c4ca3f5e3.png)

Which is kinda garbage, however we can fix this for you if required.

Reply:
Strange, I just exported this with KLE and this conversion tool. Where can I test it myself?
Here is the original: http://www.keyboard-layout-editor.com/#/gists/66284dc2636a36c6ff7d0a4669cfddcc

Reply:
Looking at it, could it be the rotation of the keys?

Reply:
Yep, the info.json format (and thus qmk.fm/converter) doesn't support rotation at present.

Reply:
Ok, I removed it and pushed a rotation-free version

Reply:
![image](https://user-images.githubusercontent.com/4781841/81852519-81023b80-959e-11ea-819e-7417f4fa5db8.png)
Looks much better 👍

Reply:
great! thanks for testing.

Reply:
Cheers for that!